continuous_integration/environment-3.7-jdk11-dev.yaml

```diff
@@ -19,6 +19,7 @@ dependencies:
 - mlflow>=1.19.0
 - mock>=4.0.3
 - nest-asyncio>=1.4.3
+- nvtx>=0.2.1
```

Comment:
We will also need to add this requirement to the [conda recipe](https://github.com/rapidsai/dask-sql/blob/branch-22.02/continuous_integration/recipe/meta.yaml) for the nightly builds to succeed

Reply:
Thanks @charlesbluca, Added it.